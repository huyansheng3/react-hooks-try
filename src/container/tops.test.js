import React from 'react';
import ReactDOM from 'react-dom';
import Tops from './tops';
import { render, fireEvent, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import MockAdapter from "axios-mock-adapter";

test('test loading status', () => {
  const { getByText, getByTestId } = render(<Tops />)

})

test('test error status', () => {

})

test('test cancel request', () => {

})

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Tops />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Tops renders snapshot', () => {
  const tree = renderer
    .create(<Tops />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});