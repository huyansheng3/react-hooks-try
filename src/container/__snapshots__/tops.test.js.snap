// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tops renders snapshot 1`] = `
<div
  style={
    Object {
      "height": "100vh",
      "padding": "30px",
    }
  }
>
  <form
    onSubmit={[Function]}
  >
    <div>
      <input
        onChange={[Function]}
        type="text"
        value="redux"
      />
      <button
        type="submit"
      >
        Search
      </button>
    </div>
  </form>
  <ul
    style={
      Object {
        "maxHeight": "300px",
        "overflowY": "scroll",
      }
    }
  />
</div>
`;
